tests/components/script/test_init.py

```diff
@@ -1502,3 +1503,34 @@ async def test_blueprint_script_fails_substitution(
         "{'service_to_call': 'test.automation'}: No substitution found for input blah"
         in caplog.text
     )
+
```

Comment:
Here is a test for the exclusive case if you want it:

```
async def test_stop_action_response_and_error_exclusive(
    hass: HomeAssistant, caplog: pytest.LogCaptureFixture
) -> None:
    """Test config validation for exvlusive fields in stop action."""
    with pytest.raises(vol.Invalid, match="two or more values in the same group of exclusion"):
        cv.SCRIPT_SCHEMA(
            [
                {
                    "alias": "stop step",
                    "stop": "In the name of love",
                    "error": True,
                    "response": "response-value",
                },
            ]
        )
```

Reply:
Added in https://github.com/home-assistant/core/pull/95376